### PR TITLE
Add tests results to html report

### LINKED ISSUE #167: Show the tests results in the html report
Show the tests results in the html report. Show if each test case passed or failed in the html reports

### LINKED ISSUE #169: Show a tests summary in the HTML report
Show a tests summary in the HTML report.

- How many tests passed
- How many tests failed
- How many tests crashed
- How long it took to run all tests

### PR DESCRIPTION
**Pull request by camilamaia**
- Show a tests summary in the HTML report
- Show each test result in the html report
- Small refactor on Reporter

Current this is how it is rendered: https://github.com/scanapi/scanapi/blob/adr2/testing_reports/current.html

report
report2

But we are still discussing better how we are going to present it #161 . We can improve it later in the next PRs.

Closes #169 
Closes #167

### CODE REVIEW

File: scanapi/template_render.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import curlify
```
Comment (gillianomenezes) [inline scanapi/template_render.py:1]: I like how this refactor follows the Single Responsibility Principle 😃

File: scanapi/reporter.py [hunk 2]
```diff
@@ -15,28 +14,25 @@ def __init__(self, output_path=None, template=None):
...
         logger.info(f"It is available at {self.output_path}")
 
-    def _render_content(self, loader, template_path, responses):
-        env = Environment(loader=loader)
-        env.filters["curlify"] = curlify.to_curl
-        env.globals["now"] = datetime.datetime.now().replace(microsecond=0)
-        env.globals["project_name"] = settings.get("project-name", "")
-        chosen_template = env.get_template(template_path)
-        return chosen_template.render(responses=responses)
+    def _build_context(self, results):
```
Comment (gillianomenezes) [inline scanapi/reporter.py:32]: Excellent refactor! It turned to be easier to read 🔍

File: scanapi/template_render.py (hunk 1 above)
Comment (gillianomenezes) [reply scanapi/template_render.py:1]: I mean the entire class

**Comment by camilamaia** [comment]
@gillianomenezes thank you very much for the comments and feedbacks!!! 🎉

**Pull request merged**